metaflow/metaflow_config.py

```diff
@@ -255,6 +255,54 @@
 for typ in DEBUG_OPTIONS:
     vars()["DEBUG_%s" % typ.upper()] = from_conf("DEBUG_%s" % typ.upper())
 
+###
+# Plugin configuration
+# List of plugins that are enabled.
```

Comment:
what is the rationale behind listing everything as enabled here? Why can't we have everything enabled by default and only disable certain things within the extensions?

Reply:
The rationale was @tuulos 's comment about having a more traditional approach where you list the plugins you want (so basically have an ENABLED_ list. However, that was a bit hard to square with the notion of "no change from what we currently have" so this is what i came up with: you do have an ENABLED list where users can explicitly select what they want but the default is "everything" (as opposed to nothing). I am fine switching to something else but that was why I did it this way. Initially, iirc, I just had a notion of toggle and changed it to this based on feedback.